app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>User posts with Id</h1>
\ No newline at end of file
+
+<% @posts.each do |post| %>
+<ul class="blog-container">
+ <li class="blog-card">
+      <a href="<%= user_post_path(post.author_id, post.id) %>" class="blog">
+        <h2><%= post.title %></h2>
+        <p><%= post.text %></p>
+        <div class="comment-like">
+        <span>Comments: <%= post.comments.length %></span>
+        <span>Likes: <%= post.likes.length %></span>
+        </div>
+      </a>
+    </li>
+    </ul>
+
+    <ul class="comment-container">
+      <% unless post.comments.any? %>
+        <h2>No Comments</h2>
+      <% end %>
+      <% post.recent_comments.each do |comment| %>
+        <li >
+          <span><%= User.find(comment.user_id).name %>:</span>
+          <span><%= comment.text %></span>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
```

Comment:
- [ ] Amazing effort so far👏✨ Please make sure that post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) ⛏ This is what we have right now 👇

![image](https://github.com/manzitresor/blog_app/assets/106119105/5a08c46f-768d-4c88-ae1b-75bebf4d4d2c)